libkbfs/md_journal_test.go

```diff
@@ -585,6 +614,8 @@ func testMDJournalBranchConversion(t *testing.T, ver MetadataVer) {
 	mdcache := NewMDCacheStandard(10)
 	cachedMd := makeMDForTest(
 		t, ver, id, firstRevision, j.uid, signer, firstPrevRoot)
+	err := cachedMd.bareMd.SignWriterMetadataInternally(ctx, codec, signer)
```

Comment:
Right, but the call in `makeMDForTest` was redundant -- `mdJournal.put` does the signing, since it calls `encryptMDPrivateData` (which does the actual signing).

But it's necessary here since we're injecting an MD into the cache.